content/en/workflows/actions_catalog/_index.md

```diff
@@ -3,11 +3,16 @@ title: Actions Catalog
 kind: documentation
 type: actioncatalog
 disable_toc: true
-further_reading:
-- link: "logs/processing/pipelines"
-  tag: "Documentation"
-  text: "Log processing pipelines"
 cascade:
   disable_sidebar: true
   type: actioncatalog
 ---
+
```

Comment:
Please add `is_beta: true` to the frontmatter, as described in docs4docs: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown#Beta-callouts.

Reply:
Thanks, added